mysql/migrations.sql

```diff
@@ -89,3 +89,19 @@ ALTER TABLE `taskHistory` ADD KEY `runId` (`runId`, `requestId`);
 ALTER TABLE `requestHistory` ADD COLUMN message VARCHAR(280) NULL;
 
 ALTER TABLE `deployHistory` ADD COLUMN message VARCHAR(280) NULL;
+
+--changeset wsorenson:10 dbms:mysql
+ALTER TABLE `taskHistory`
+  ADD COLUMN `host` VARCHAR(100) CHARACTER SET ASCII NULL,
+  ADD COLUMN `startedAt` TIMESTAMP NULL,
```

Comment:
Yep, we are. I was thinking about this in the context of pagination -- `LIMIT :offset, :count` isn't great the further in you go and since the offset isn't a concrete point in time, runs of the persister will make the results randomly update on you. Paging based on `startedAt` would be nice if it was more accurate. Maybe we could look into an auto_increment id? This isn't a super high priority thing, of course, but it might be worth thinking about some more in the future.